app/Models/Post.php

```diff
@@ -167,7 +195,7 @@ public function getMediaUrl()
             return null;
         }
 
-        return url('images/' . $this->id);
+        return url('images/' . $this->hash);
```

Comment:
I would have expected an update for when we first store the image to use this as well?

Reply:
Whatcha mean?

Reply:
Great question let me try to rephrase - when do we store the image at `'images/' . $this->hash`?

Reply:
Ah, we don't! This is our [Glide](https://glide.thephpleague.com) route that renders images for use in galleries (or wherever). It used to use the post's ID to load the database record (and from that, the actual S3 file path), but this PR swaps it to use this predictable hash instead so that someone can't just iterate over every image and find unapproved ones.

Reply:
Correct!

Reply:
ohhhhhhhh 💡